src/main/resources/static/swagger.yaml

```diff
@@ -89,7 +89,28 @@ paths:
           description: Notification successful.
         '400':
           description: Unauthorized.
-
+  /health-information/fetch/{consent-request-id}/:
```

Comment:
do you mean, it would fetch all the data belonging to a consent-requests, which means - if for a given request-id, there were 3 consent artefacts, then it would fetch all the health data belonging across the consent-artefacts?

Reply:
Yes, thats the reason we have it for each HIP Id.